### PR TITLE
windows-app 11.0.1 (new cask)

### PR DESCRIPTION
**Pull request by yurikoles**
Signed-off-by: Yurii Kolesnykov <root@yurikoles.com>

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: cask_renames.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
   "hma-pro-vpn": "hma-vpn",
   "mat": "memoryanalyzer",
   "marathon2": "marathon-2",
+  "microsoft-remote-deskto": "windows",
```
Comment (bevanjkay) [inline cask_renames.json:31]:
```suggestion
  "microsoft-remote-desktop": "windows",

**Comment by bevanjkay** [comment]
Thank you @yurikoles 
I think we would be best keeping the old cask around for now, and adding `windows` as a new cask.

**Comment by MTCoster** [comment]
I know the naming rules would usually require this to be called `windows`, but would it make sense for an exception to be made here? It definitely feels to me like `windows-app` would be more appropriate (although I 100% blame Microsoft for the terrible name here).

**Comment by yurikoles** [comment]
Installing Windows 11 via homebrew feels weird for me as well. 😅

**Comment by yurikoles** [comment]
Let's wait for the maintainers' opinion.

**Comment by bevanjkay** [comment]
I am okay to go with `windows-app` instead here as I think it does help with the disambiguation.

**Comment by yurikoles** [comment]
Renamed.

File: Casks/w/windows-app.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+cask "windows-app" do
+  version "11.0.1"
+  sha256 "8c53f6bc8f863c984967193c4f7572be80e0154cf90aa2db22303edcc3172faa"
+
+  url "https://officecdnmac.microsoft.com/pr/C1297A47-86C4-4C1F-97FA-950631F94777/MacAutoupdate/Windows_App_#{version}_installer.pkg",
+      verified: "officecdnmac.microsoft.com/pr/C1297A47-86C4-4C1F-97FA-950631F94777/MacAutoupdate/"
+  name "Windows App"
+  desc "Connect to Windows"
```
Comment (khipp) [inline Casks/w/windows-app.rb:8]: This is a mere suggestion, but I was trying to come up with a more fitting description.

How about:

```suggestion
  desc "Connect to Windows desktops and applications"
```

or:

```suggestion
  desc "Connect to Windows devices and services"
```

Comment (yurikoles) [reply Casks/w/windows-app.rb:8]: I took the descriptions from the [App Store tagline](https://aka.ms/macOSWindowsApp).

**Review by bevanjkay — APPROVED** [review]

**Comment by VyacheslavPritykin** [comment]
This cask is described as `"auto_updates": true` and it installs `com.microsoft.update.agent` with `com.microsoft.autoupdate.helper`, however,  Microsoft AutoUpdate (MAU) does not want either to see or update the Windows App:

image

I suggest removing the `"auto_updates": true` for now.

**Comment by yurikoles** [comment]
#187939

**Pull request merged**